src/identity/interaction/email-password/util/RegistrationManager.ts

```diff
@@ -139,7 +140,7 @@ export class RegistrationManager {
     // Parse WebID
     if (!validated.createWebId) {
       const trimmedWebId = this.trimString(webId);
-      assert(trimmedWebId && /^https?:\/\/[^/]+/u.test(trimmedWebId), 'Please enter a valid WebID.');
+      assert(trimmedWebId && hasScheme(trimmedWebId, 'http', 'https'), 'Please enter a valid WebID.');
```

Comment:
Makes me realize this can also be done here: https://github.com/CommunitySolidServer/CommunitySolidServer/blob/dd568869ca2228e3987c9fd998c778e181ada2bb/src/identity/storage/WebIdAdapterFactory.ts#L45